Sending authorisation email fails if no "person" attached to event

@johnathan99j found this with event 3930

If there is no "person" attached to an event, clicking the "send authorisation email" fails silently. This is the error in the server logs:

```
2017-12-01T12:13:50.477186+00:00 app[web.1]: 2017-12-01 12:13:32 [8] [ERROR] pathname=/app/.heroku/python/lib/python3.6/site-packages/django/core/handlers/exception.py lineno=135 funcname=handle_uncaught_exception Internal Server Error: /event/3930/auth/
2017-12-01T12:13:50.477199+00:00 app[web.1]: Traceback (most recent call last):
2017-12-01T12:13:50.477200+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/core/handlers/exception.py", line 41, in inner
2017-12-01T12:13:50.477201+00:00 app[web.1]:     response = get_response(request)
2017-12-01T12:13:50.477202+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
2017-12-01T12:13:50.477203+00:00 app[web.1]:     response = self._get_response(request)
2017-12-01T12:13:50.477203+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/core/handlers/base.py", line 187, in _get_response
2017-12-01T12:13:50.477204+00:00 app[web.1]:     response = self.process_exception_by_middleware(e, request)
2017-12-01T12:13:50.477205+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/core/handlers/base.py", line 185, in _get_response
2017-12-01T12:13:50.477206+00:00 app[web.1]:     response = wrapped_callback(request, *callback_args, **callback_kwargs)
2017-12-01T12:13:50.477207+00:00 app[web.1]:   File "/app/PyRIGS/decorators.py", line 25, in _checklogin
2017-12-01T12:13:50.477207+00:00 app[web.1]:     return view_func(request, *args, **kwargs)
2017-12-01T12:13:50.477208+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
2017-12-01T12:13:50.477210+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/utils/decorators.py", line 67, in _wrapper
2017-12-01T12:13:50.477209+00:00 app[web.1]:     return self.dispatch(request, *args, **kwargs)
2017-12-01T12:13:50.477210+00:00 app[web.1]:     return bound_func(*args, **kwargs)
2017-12-01T12:13:50.477211+00:00 app[web.1]:   File "/app/PyRIGS/decorators.py", line 93, in wrap
2017-12-01T12:13:50.477211+00:00 app[web.1]:     return function(request, *args, **kwargs)
2017-12-01T12:13:50.477212+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/utils/decorators.py", line 63, in bound_func
2017-12-01T12:13:50.477213+00:00 app[web.1]:     return func.__get__(self, type(self))(*args2, **kwargs2)
2017-12-01T12:13:50.477213+00:00 app[web.1]:   File "/app/RIGS/rigboard.py", line 309, in dispatch
2017-12-01T12:13:50.477214+00:00 app[web.1]:     return super(EventAuthorisationRequest, self).dispatch(*args, **kwargs)
2017-12-01T12:13:50.477215+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/views/generic/base.py", line 88, in dispatch
2017-12-01T12:13:50.477216+00:00 app[web.1]:     return handler(request, *args, **kwargs)
2017-12-01T12:13:50.477216+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/views/generic/edit.py", line 183, in post
2017-12-01T12:13:50.477217+00:00 app[web.1]:     return self.form_valid(form)
2017-12-01T12:13:50.477217+00:00 app[web.1]:   File "/app/RIGS/rigboard.py", line 343, in form_valid
2017-12-01T12:13:50.477218+00:00 app[web.1]:     if email == event.person.email:
2017-12-01T12:13:50.477222+00:00 app[web.1]: AttributeError: 'NoneType' object has no attribute 'email'
```
We should either:
- Not allow sending authorisations for events without a person
or 
- Use the organisation email address if there is no person

--- a/RIGS/rigboard.py
+++ b/RIGS/rigboard.py
@@ -340,8 +340,10 @@ class EventAuthorisationRequest(generic.FormView, generic.detail.SingleObjectMix
                 'sent_by': self.request.user.pk,
             }),
         }
-        if email == event.person.email:
+        if event.person is not None and email == event.person.email:
             context['to_name'] = event.person.name
+        elif event.organisation is not None and email == event.organisation.email:
+            context['to_name'] = event.organisation.name
 
         msg = EmailMultiAlternatives(
             "N%05d | %s - Event Authorisation Request" % (self.object.pk, self.object.name),

